raiden/network/proxies/token_network.py

```diff
@@ -234,6 +241,12 @@ def new_netting_channel_light(self, creator: Address, partner: Address, signed_t
                     log.critical("new_netting_channel_light failed", **log_details)
                     raise RaidenRecoverableError("creating new channel failed")
             except HTTPError as e:
+                self.proxy.jsonrpc_client.check_for_insufficient_eth(
+                    transaction_name="openChannel",
+                    address=creator,
+                    transaction_executed=False,
+                    required_gas=GAS_REQUIRED_FOR_OPEN_CHANNEL
+                )
                 log.warning("new_netting_channel failed: transaction malformed", ex=e, **log_details)
                 new_open_channel_transaction.set_exception(e)
```

Comment:
it line relevant? 

note that if the check_for_insufficient_eth raises insufficient funds, this will not be set

im not familiar with this, the set exception basically does a notifier/subscribe machanism,  not sure if we are subscribing on any part of the system to that events. I dont think so:

```    
def set_exception(self, exception, exc_info=None):
        """Store the exception and wake up any waiters.

        All greenlets blocking on :meth:`get` or :meth:`wait` are awakened.
        Subsequent calls to :meth:`wait` and :meth:`get` will not block at all.

        :keyword tuple exc_info: If given, a standard three-tuple of type, value, :class:`traceback`
            as returned by :func:`sys.exc_info`. This will be used when the exception
            is re-raised to propagate the correct traceback.
        """

Reply:
Probably you are right i will move the call after this because we don't know